_posts/2021-04-17-mapping-tutorial.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: CargoRocket Mapathon Anleitung
+author: Henri Chilla
+excerpt: Eine Anleitung zum erfolgreichen Mappen für den CargoRocket Mapathon.
+description: Tutorial zum CargoRocket Mapathon
+---
+
+Um an unserem Mapathon teilzunehmen, ist nicht viel notwendig! Hier gehen wir kurz darauf ein, wie an dem Datensammeln mitgewirkt werden kann.
+
+## OpenStreetMap
+Für einen guten Index und ein gutes Routing sind gute Daten notwendig. Hierfür beziehen wir unsere Daten von [OpenStreetMap](https://www.openstreetmap.org) (OSM). An diesem Open-Source Projekt kann sich nach dem Wikipedia-Prinzip jede Person beteiligen und beliebige Daten eintragen. Dementsprechend ist es möglich, wichtige Daten für Lastenräder zu erfassen. Diese nützen dann aber auch allen anderen Nutzer:innen, die auf die Daten zugreifen, bspw. auch Rollstuhlfahrenden.
+
+## Einführung OSM
+*Wenn du bereits bei OSM aktiv bist, kannst du diesen Abschnitt vermutlich überspringen.*
+Für das Eintragen der Daten wird zunächst ein OSM-Nutzerkonto benötigt, welcher [hier](https://www.openstreetmap.org/user/new) erstellt werden kann. Für das Bearbeiten existieren verschiedenste Editoren. Für den Einstieg sind folgende am besten geeignet: [ID-Editor](https://learnosm.org/de/beginner/id-editor/) und [StreetComplete](https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete). Für unterwegs auf dem Smartphone existieren auch [Vespucci](https://play.google.com/store/apps/details?id=de.blau.android&hl=de&gl=US) (Android) und [Go Map!!](https://apps.apple.com/de/app/go-map/id592990211) (iOS) um eine Auswahl zu erwähnen.
+
+Eine allgemeine Einführung gibt es bei [LearnOSM](https://learnosm.org/de/), denn dies soll keine Anleitung für OSM sein.
+
+## Tags
+Wir haben verschiedene Faktoren identifiziert, welche wichtig für Lastenrad-GIS-Applikationen sind. Daraus haben wir dann die entsprechenden OSM-Tags abgeleitet. Für unsere Auswertungen verwenden wir folgende generellen Tags:
+
+* Radwege
+  *  [`surface=*`](https://wiki.openstreetmap.org/wiki/DE:Key:surface)
+  *  [`smoothness=*`](https://wiki.openstreetmap.org/wiki/DE:Key:smoothness)
+  *  [`width=*`](https://wiki.openstreetmap.org/wiki/DE:Key:width)
+*  Poller
+   *  [`maxwidth:physical=*`](https://wiki.openstreetmap.org/wiki/DE:Key:maxwidth:physical)
+
+> Vorsicht: Es gibt allerdings viele verschiedene Möglichkeiten und entsprechende [Sub-Tags (Namespaces)](https://wiki.openstreetmap.org/wiki/DE:Attribut#Schl.C3.BCssel_und_Werte), die bspw. den Radstreifen neben der Fahrbahn (`cycleway:right=lane`). Diese werden bevorzugt, wenn vorhanden.
+
+## Beispiel Szenarien
+
+### Eigener Radweg
```

Comment:
Vielleicht das als Beispielbild? https://drive.google.com/file/d/1kb58eejB77psl5ldMcQS9XhT_ZPq9SKb/view?usp=sharing

Reply:
Mhm vorläufig wäre das passend! Wollte am besten noch eins mit Zeichen 237 (Radweg) 
![grafik](https://user-images.githubusercontent.com/6567206/114668484-17843300-9d01-11eb-8551-e45a8ec99494.png)

Aber dann integriere ich das erstmal so!

Reply:
ah I see. Ich schau mal ob ich an einem vorbei fahre die Tage. Sonst ist "designated" oder nicht für uns ja eigentlicih gar nicht so relevant, oder?

Reply:
wie wärs damit? https://drive.google.com/file/d/17ojUjZVwPrCuM7dbI9KgJ6JkbXQKOlWb/view?usp=sharing

Reply:
Danke, optimal! Was genau meinst du mit "designated"? highway=path bicycle=designated?

Reply:
Ja genau. Ich hab das so verstanden, dass "designated" bedeutet, ob es (u.a. durch das blaue Schild) rechtsverbindlich ist, oder nur "empfholen" aber man als Radfahrer*in auch auf der Straße fahren darf. 
Das berücksichtigen wir (zumindest bisher) ja nicht